apps/web/src/views/Swap/V3Swap/containers/FormMain.tsx

```diff
@@ -30,9 +30,17 @@ interface Props {
   tradeLoading?: boolean
   pricingAndSlippage?: ReactNode
   swapCommitButton?: ReactNode
+  loadedUrlParams: boolean
```

Comment:
Can we rename the variable for better understanding?

```suggestion
  isUrlParamsLoaded: boolean
```